hashintel/hash
# Implement resizable columns on grid

## 🌟 What is the purpose of this PR?

This PR lets users resize column sized to be able to change column widths as they wish for UX.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203271043333970/1203267989886076/f) _(internal)_


## 🔍 What does this change?

- Adds an optional `resizable` property to `GlideGrid` component to enable resizable columns

## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Open entity editor
1.  Confirm that columns on property table are resizable (value column cannot make smaller, because it has `grow: 1`, but you should be able to resize other two columns)

## 📹 Demo

https://user-images.githubusercontent.com/39553853/201876508-022f07a7-0139-4443-88b4-94605fca11c2.mov
